components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa/src/Publisher/Models/DataSetContentMask.cs

```diff
@@ -50,6 +50,36 @@ public enum DataSetContentMask {
         /// <summary>
         /// Sequence number
         /// </summary>
-        SequenceNumber = 0x80
+        SequenceNumber = 0x80,
+
+        /// <summary>
+        /// Message type (json)
+        /// </summary>
+        MessageType = 0x100,
+
+        /// <summary>
+        /// Dataset writer name (json)
+        /// </summary>
+        DataSetWriterName = 0x200,
+
+        /// <summary>
+        /// Reversible encoding (json)
+        /// </summary>
+        ReversibleFieldEncoding = 0x400,
+
+        /// <summary>
+        /// Default non-reversible json
+        /// </summary>
+        DefaultJson = MessageType | DataSetWriterName |
+            SequenceNumber | DataSetWriterId |
+            Timestamp | MetaDataVersion | Status,
+
+        /// <summary>
+        /// Default Uadp
+        /// </summary>
+        DefaultUadp = MinorVersion | SequenceNumber |
```

Comment:
`MinorVersion` seem to be duplicate. 
I'm not sure I understand why DefaultUdp is different than the DefaultJson.

Reply:
There are different flags valid for UADP vs. Json.  E.g., UADP allows encoding just major or minor, while JSON requires both.